chime-notifier/README.md

```diff
@@ -0,0 +1,51 @@
+## AWS Health Chime Notifier
+
+### Description
+
+This tool can be used to post alerts to a Chime room when AWS Health events are generated by using AWS Lambda and Amazon CloudWatch Events. 
+
+### Chime Setup
+Follow these steps to configure the webhook in Chime:
+
+1. Click the cog wheel from your Chime room 
+
+2. Select "Manage WebHooks".
+
+3. Create Webhook
+
+4. Copy the webhook URL and use it in the AWS Setup section that follows.
+
+
+### AWS setup using CloudFormation 
+
+#### CloudFormation
+Choose **Launch Stack** to launch the template in the US East (N. Virginia) Region in your account:
+ 
+
+ 
+The CloudFormation template requires the following parameter:
+ 
+*HookURL* - Incoming web hook url from Slack setup.
+
+
+### AWS Manual Setup
+
+1. Create an IAM role for the Lambda function to use. Attach the [IAM policy](IAMPolicy) to the role in the IAM console.
+Documentation on how to create an IAM policy is available here: http://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html
+Documentation on how to create an IAM role for Lambda is available here: http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html#roles-creatingrole-service-console
+
+2. Create a Lambda Python function by using the [sample](LambdaFunction.py) provided and choose the IAM role created in step 1. Add an enviroment variable with key CHIMEWEBHOOK and the webhook URL from the Chime setup above as value.
+
+More information about Lambda is available here: http://docs.aws.amazon.com/lambda/latest/dg/getting-started.html
+
+3. Create a CloudWatch Events rule to trigger the Lambda function created in step 2 for AWS Health events.
+Documentation on how to create an AWS Health CloudWatch Events rule is available here: http://docs.aws.amazon.com/health/latest/ug/cloudwatch-events-health.html
+
+More information about AWS Health is available here: http://docs.aws.amazon.com/health/latest/ug/what-is-aws-health.html
+
+Note that this is a just an example of how to set up automation with AWS Health, Amazon CloudWatch Events, and AWS Lambda. We recommend testing the example and tailoring it to your environment before using it in your production environment.
+
```

Comment:
Do you have a sample test you could suggest to the user about how they could force a health event?
At a minimum there could be a link to the documentation at http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html#health-event-types which would show a structure for doing a test of their Lambda configuration at least.

Reply:
Updated README.md with clarification on how to manually test and a link to the Health test event page.